rust/Cargo.toml

```diff
@@ -112,6 +116,7 @@ paste = "1.0"
 pretty_env_logger = "0.5.0"
 primitive-types = "=0.12.1"
 prometheus = "0.13"
+protobuf = "2.28.0"
```

Comment:
we already have protobuf in cargo lock so I'd also try resolving it with a wildcard here

Reply:
nice this worked too